### PR TITLE
add wordpress integration tests for php images

### PR DESCRIPTION
**Pull request by arroyc**
php integration tests for WordPress sample apps

### CODE REVIEW

File: tests/Oryx.Integration.Tests/Php/PhpWordPressTest.cs [hunk 1]
```diff
@@ -68,5 +68,128 @@ public async Task WordPress51(string phpVersion)
...
+                    Assert.Contains("Remember Me", data);
+                    Assert.Contains("Lost your password?", data);
+                    Assert.Contains("Back to localsite", data);
+                });
+        }
+
+        [Theory]
+        [InlineData("7.4")]
+        [InlineData("7.3")]
+        [InlineData("7.2")]
```
Comment (qianz2) [inline tests/Oryx.Integration.Tests/Php/PhpWordPressTest.cs:136]: Maybe add 7.0, 5.6 version as well?

File: tests/Oryx.Integration.Tests/Php/PhpWordPressTest.cs [hunk 2]
```diff
@@ -68,5 +68,128 @@ public async Task WordPress51(string phpVersion)
...
+                .AddCommand($"cd {appDir}")
+                .AddFileExistsCheck("oryx-manifest.toml")
+                .AddFileExistsCheck("wp-cli.phar")
+                .AddFileExistsCheck("create_wordpress_db.sh")
+                .AddFileExistsCheck("configure_wordpress.sh")
+                .AddCommand($"chmod +x create_wordpress_db.sh && ./create_wordpress_db.sh")
+                .AddCommand($"chmod +x configure_wordpress.sh && ./configure_wordpress.sh")
+                .AddCommand($"oryx create-script -appPath {appDir} -bindPort {ContainerPort} -output {RunScriptPath}")
+                .AddCommand("mkdir -p /home/site/wwwroot")
+                .AddCommand($"cp -a {appDir}/. /home/site/wwwroot")
```
Comment (qianz2) [inline tests/Oryx.Integration.Tests/Php/PhpWordPressTest.cs:168]: will `cp -rf` be better instead of `cp -a`?

Comment (arroyc) [reply tests/Oryx.Integration.Tests/Php/PhpWordPressTest.cs:168]: I tried with cp -rf first and it was failing so trying with cp -a, as per doc cp -a seems to be a better version of cp -rf

File: tests/Oryx.Integration.Tests/Php/PhpWordPressTest.cs (hunk 1 above)
Comment (arroyc) [reply tests/Oryx.Integration.Tests/Php/PhpWordPressTest.cs:136]: we dont support php fpm images of 7.0 and 5.6. those are only supported in regular php images that has apache

File: tests/Oryx.Integration.Tests/Php/PhpWordPressTest.cs (hunk 2 above)
Comment (qianz2) [reply tests/Oryx.Integration.Tests/Php/PhpWordPressTest.cs:168]: gotcha, thanks

**Comment by kichalla** [comment]
Make sure the Nightly passes before checkin

**Pull request merged**